### ISSUE TITLE
Troops not able to deploy into buildings

### ISSUE THREAD

**Issue by adp50** [OpenRA/ra2#204]
In the original RA2, you could station your troops inside the buildings. With the troops selected, when hovering the cursor above a building, the cursor would turn into an arrow, allowing you to station troops inside the building. There was also an indicator next to the building which showed how many troops were currently inside, and available spaces.

**Comment by GraionDilach** [comment]
Yes, we're aware. Half of this logic already exists in OpenRA, but the crucial "allow-entering-to-neutral-structures-to-temporaly-gain-ownership" isn't, which is why it's missing atm.

**Comment by adp50** [comment]
Thanks for the update. Would there be any approximate timescale for this to be implemented into OpenRA?

**Comment by GraionDilach** [comment]
There isn't. RA2 logics are on really low priority. This is a third-party mod afterall.

**Comment by adp50** [comment]
I understand. Great to know it is in the pipeline for the future anyway.

**Comment by ibrokemypie** [comment]
+1

**Comment by Mailaender** [comment]
https://www.moddb.com/mods/romanovs-vengeance already has it. Needs a backport.
